docs/reference/commandline/version.md

```diff
@@ -18,38 +18,97 @@ Options:
 
 ## Description
 
-By default, this will render all version information in an easy to read
-layout. If a format is specified, the given template will be executed instead.
+The `version` command prints detailed information about the version of the
+`docker` CLI, the Docker Engine (`dockerd`), as well as other components.
 
-Go's [text/template](https://golang.org/pkg/text/template/) package
```

Comment:
I think this might be a good xref to use here https://docs.docker.com/config/formatting/